### DISCUSSION TITLE
MultiSelect 在表格的编辑弹窗中更新问题

### DISCUSSION THREAD

**Discussion by zhangyu-roger** [dotnetcore/BootstrapBlazor#3491] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

如题，多项选择器**在表格的弹出窗口中**，勾选或取消选项时刷新有问题，表现为：
1. 勾选后，MultiSelect 的编辑框没有显示
2. 取消勾选或是点击标签的叉关闭选项时，MultiSelect 的编辑框没有显示
3. 再次打开编辑弹窗时，上次的修改生效了。
4. 同样的设置在 单项选择器 中没有问题
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/159638053/ceee636b-b576-48de-9991-e7da75acb6a4)

**Comment by bb-auto** [comment]
@zhangyu-roger Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by ArgoZhang** [comment]
@zhangyu-roger 无效 Issue 请按照格式提供信息，一天内不更新，自动关闭

**Comment by zhangyu-roger** [comment]
这个是最小化的重现包
[MSelectInGridEditor.zip](https://github.com/dotnetcore/BootstrapBlazor/files/15256463/MSelectInGridEditor.zip)
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/159638053/da1f0213-e23d-4572-82e5-8b85949ddaf6)

**Reply by ArgoZhang** [reply]
@zhangyu-roger 右上角项目帮忙点一下 star 谢谢啦
